docs/installation/micro-rdk-dev.md

```diff
@@ -225,7 +225,8 @@ Instead, save this command to run in any future terminal session where you need
 Navigate to [the Viam app](https://app.viam.com) and [add a new machine](/cloud/machines/#add-a-new-machine) in your desired location.
 
 Click on the name of the machine to go to the machine's page.
-Then, navigate to the **CONFIGURE** tab and [configure your machine](/configure/).
+Then, navigate to the **CONFIGURE** tab and [configure your machine](/configure/) with an `esp32` board and any components you want.
+You can also do this later.
```

Comment:
I actually don't think they should do that at this stage. Once connected they can actually test. So makes more sense to complete setup first. 
```suggestion
```